vue-frontend/src/components/jobs/LifeAtCanopas.vue

```diff
@@ -21,7 +21,7 @@
         :lazy="true"
         :autoplay="{
           delay: 3000,
-          disableOnInteraction: true,
+          disableOnInteraction: false,
```

Comment:
Why its false?

Reply:
when we click on swiper-pagination bullet with mouse autoplay gets stop. to solve this i have done false